### DISCUSSION TITLE
Straight lines without pressing shift

### DISCUSSION THREAD

**Discussion by k777ii** [Softal-io/presentify-community#5] [Ideas]
can add a feature to do either "freehand" or straight lines without pressing shift key on the mac?

**Comment by rampatra** [comment]
Will add a Setting in the Preferences to set the default free hand mode. If it’s set to straight lines then pressing and holding shift would disable it. 

Sound good?

**Comment by k777ii** [comment]
OkSent from my iPhoneOn Jun 24, 2023, at 3:15 PM, Ram ***@***.***> wrote:﻿
Will add a Setting in the Preferences to set the default free hand mode. If it’s set to straight lines then pressing and holding shift would disable it.
Sound good?

**Comment by k777ii** [comment]
Better to add a features to use free hand or active it free hand or straight line, like on and off switch. Sent from my iPhoneOn Jun 24, 2023, at 3:15 PM, Ram ***@***.***> wrote:﻿
Will add a Setting in the Preferences to set the default free hand mode. If it’s set to straight lines then pressing and holding shift would disable it.
Sound good?

**Comment by rampatra** [comment]
Okay, will give some thought and comment here later.

**Comment by miiraheart** [comment]
is it possible that when we draw a line and then without releasing: keep holding the line at the end would make the line straight, same as with iPad and pencil ?
